app/src/main/java/lt/vilnius/tvarkau/API/GetNewProblemParams.java

```diff
@@ -0,0 +1,32 @@
+package lt.vilnius.tvarkau.API;
+
+import android.support.annotation.Nullable;
+
+public class GetNewProblemParams {
+
+    private String session_id;
+    private String description;
+    private String type;
+    private String address;
+    private double x;
+    private double y;
+    private String[] photo;
+    private String email;
+    private String phone;
+    private String message_description;
+
+    public GetNewProblemParams(String session_id, String description, String type, String address, double latitude,
+        double longitude, @Nullable String[] photo, @Nullable String email, @Nullable String phone,
+        @Nullable String messageDescription) {
```

Comment:
When there are that many parameters, constructors become hardly usable. Better to use Builder pattern then.

Btw, IDE should be able to generate that, found this: https://www.jetbrains.com/help/idea/2016.2/replace-constructor-with-builder.html